packages/core/core/src/Transformation.js

```diff
@@ -470,6 +470,7 @@ export default class Transformation {
       pipeline: a.value.pipeline,
       hash: a.value.hash,
       uniqueKey: a.value.uniqueKey,
+      query: a.value.query,
```

Comment:
Do we need to sort this? Maybe we should sort the query params in one place too so we don't need to keep sorting it?

Reply:
Maybe, although every transformer can modify the query params so maybe it's not a good idea?
Not sure how much value it would add